src/content/docs/en/recipes/build-forms.mdx

```diff
@@ -8,6 +8,10 @@ import { Steps } from '@astrojs/starlight/components';
 
 Astro pages that are rendered on demand can both display and handle forms. In this recipe, you'll use a standard HTML form to submit data to the server. Your frontmatter script will handle the data on the server, sending no JavaScript to the client.
 
+:::tip[Build forms with Astro actions]
```

Comment:
"Astro Actions" is capitalized in the main guide, so I generally think we should keep that convention when talking about the feature's "brand name"
```suggestion
:::tip[Build forms with Astro Actions]
```

Reply:
Ah, I forgot we let Ben get away with this. Usually our features are not capitalized, only APIs and components. But you're right: we do capitalize when we say Astro Actions in full. (And otherwise, just "actions" when its on its own)